cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/ConfigurationTable.java

```diff
@@ -50,7 +51,7 @@
   }
 
   // since this will be used in the coprocessor context, we use commons logging
-  private static final Log LOG = LogFactory.getLog(ConfigurationTable.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ConfigurationTable.class);
```

Comment:
Components that are used by the HBase coprocessors (hence run in HBase) intentionally use commons logging, not slf4j.  This means you can easily configure their logging output using the HBase log4j config.